docs/filterable-lists.md

```diff
@@ -0,0 +1,66 @@
+
+#### EnforcementActionsFilterForm
+
+The EnforcementActionsFilterForm is an extension of FilterableListForm, adding on two fields specific to Enforcement Actions, and using a refined search class to provide search functionality against the new fields and a proper ordering by initial filing date.
+
+#### EventArchiveFilterForm
+
+The EventArchiveFilterForm is another extension of FilterableListForm, the only real modification being the invokation of an event specific search class that allows us to provide filtering based on event dates rather than page publication dates.
+
+### Documents
+
+There is currently only one type of document defined `FilterablePagesDocument` which is based off the `AbstractFilterPage` class. This document is responsible for housing data related to any of the filterable page types that extend AbstractFilterPage, including EnforcementActions, BlogPage, EventPage, and NewsRoomPage to name a few. In order to get fields that are specific to a page type, such as the status list for an Enforcement Action, you use the `prepare_field` function syntax. The use of `get_instances_from_related` is to enforce the auto updating of our index when changes occur to a specific page we have indexed rather than just the relation to `AbstractFilterPage` that is reflected in the database.
+
+### Search
+
+Search is the final piece of the puzzle, where we actually leverage a couple elasticsearch features to filter and match documents and return them in an ordered queryset. Before breaking down the search classes, its important to discuss the current implementation from an Elasticsearch perspective to understand how we're gathering results.
+
+The expanded search for filterable lists is using a [multi-match](https://www.elastic.co/guide/en/elasticsearch/reference/current/query-dsl-multi-match-query.html) across the title, topic name, preview description, and content fields of all FilterablePagesDocument. We are leveraging a [phrase_prefix](https://www.elastic.co/guide/en/elasticsearch/reference/current/query-dsl-multi-match-query.html#type-phrase) matching style with a currently configured slop of 2, to allow for some looser matching restrictions. We also provide a boost score for matching to the title and topic name fields indicated with `^10` within the code base. This boost score is to enable better ordering by relevance when desired. Search currently supports two different methods of ordering results, relevance and date published. Relevance is calculated by the Elasticsearch engine when returning results, and the date published is calculated based on page publication date. Enforcement Actions define their own ordering logic based on initial filing date. 
```

Comment:
Is "expanded search" a term of art? Not fully sure I understand what that means.

```suggestion
The expanded search for filterable lists is using a [multi-match query](https://www.elastic.co/guide/en/elasticsearch/reference/current/query-dsl-multi-match-query.html) across the title, topic name, preview description, and content fields of all `FilterablePagesDocument`s. We are leveraging a [phrase_prefix](https://www.elastic.co/guide/en/elasticsearch/reference/current/query-dsl-multi-match-query.html#type-phrase) matching style with a currently configured slop of 2, to allow for some looser matching restrictions. We also provide a boost score for matching to the title and topic name fields, indicated by `^10` within the code base. This boost score is to enable better ordering by relevance when desired. Search currently supports two different methods of ordering results: relevance and date published. Relevance is calculated by the Elasticsearch engine when returning results, and the date published is calculated based on page publication date. Enforcement Actions define their own ordering logic based on initial filing date. 
```

Reply:
Also on this paragraph:
1. I don't see where the preview description, and content fields are configured for `FilterablePagesDocument`.
2. I also don't see the `^10` indicator in the codebase. I see `boost` params of 10 on non-`FilterablePagesDocument` document models, but not on `FilterablePagesDocument` itself.

Reply:
@alexm118 Sorry to bug you, but if you have a minute to give a quick response to these questions, I'd be much obliged!

Reply:
Sorry! Just saw these. 

1. Expanded search was the term we were using to discusses enhancing the search across multiple fields. It doesn't have any specific meaning within elasticsearch.

2. [Boosting within the query](https://github.com/cfpb/consumerfinance.gov/pull/6320/files#diff-e67e781c1ded8ed3bb0a2b0c49b7f1664beb0b8678a9b05b20c0cefc4c605e3eR156) was implemented on #6320 which looks like it was just merged a few days ago.

Reply:
Thanks so much for clarifying, Alex!